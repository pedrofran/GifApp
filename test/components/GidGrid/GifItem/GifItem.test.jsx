import { render, screen } from "@testing-library/react"
import { GifItem } from "../../../../src/components/GidGrid/GifItem/GifItem";

describe('Pruebas en <GiftItem.jsx/>', () => { 
    const title = "titulo"
    const url = "https://www.google.com/"

    test('debe hacer match con el snapshot', () => {   
        const { container } = render(<GifItem title={title} url={url} />)
        expect(container).toMatchSnapshot();
     }) 

     test('la imagen debe tener un url', () => {  

      render(<GifItem title={title} url={url} />)
      
      const {src, alt} = screen.getByRole('img')

      expect( src).toBe( url )

      expect( alt).toBe( title )
      
   })

 })