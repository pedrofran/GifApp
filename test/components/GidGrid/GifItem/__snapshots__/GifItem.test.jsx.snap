// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GiftItem.jsx/> debe hacer match con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="titulo"
      src="https://www.google.com/"
    />
    <p>
      titulo
    </p>
  </div>
</div>
`;
